### PR TITLE
test: Add more cmd tests

### PR DESCRIPTION
**Pull request by yevgenypats**
#### Summary

Adding more tests. and also moving cmd tests to run via rootCmd so we can get full coverage and know that the basic commands are working. 

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [ ] Test locally on your own infrastructure
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation)) 
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

File: cli/cmd/doc_test.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+func TestDoc(t *testing.T) {
+	tmpdir, tmpErr := os.MkdirTemp("", "docs_*")
+	if tmpErr != nil {
+		t.Fatalf("failed to create temporary directory: %v", tmpErr)
+	}
+	defer os.RemoveAll(tmpdir)
+	cmd := newCmdRoot()
+	cmd.SetArgs([]string{"doc", tmpdir})
```
Comment (hermanschaaf) [inline cli/cmd/doc_test.go:28]: I didn't know about the `doc` command, should it be listed in the list of available commands?

```
Usage:
  cloudquery [command]

Available Commands:
  completion  Generate the autocompletion script for the specified shell
  generate    Generate initial config file for source and destination plugins
  sync        Sync resources from configured source plugins to destination
```

Comment (yevgenypats) [reply cli/cmd/doc_test.go:28]: I think it shouldn't because it's mainly for our website. I don't see a user using this

Comment (hermanschaaf) [reply cli/cmd/doc_test.go:28]: Yeah me neither... but in that case we should probably find a better way of generating the files for the website, no need to ship it with our cli.

Comment (hermanschaaf) [reply cli/cmd/doc_test.go:28]: (not a priority, just a side comment)

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**